include/odp/api/spec/ipsec.h

```diff
@@ -242,6 +242,11 @@ typedef struct odp_ipsec_capability_t {
 	 *  packets in ESP tunnel mode */
 	odp_bool_t tfc_padding_truncate;
 
+	/** In INLINE mode drop TFC dummy packets (NH=59) instead of reporting
+	 *  them via tfc_dummy_packet error bit.
+	 */
+	odp_bool_t tfc_dummy_packets_drop;
```

Comment:
Also this capa is not strictly needed. A portable application will need to handle also the case where inline input does not drop NH packets.

Reply:
True.